docs/getting_started/go/run_workers_with_cloud_go/index.md

```diff
@@ -40,11 +40,20 @@ They are required for configuring mutual Transport Layer Security (mTLS) protoco
 You have a few options to generate certificates.
 
 - Use existing certificate management infrastructure to generate certificates for your Namespace.
-- Use Temporal's built-in certificate generation tool [tcld](https://docs.temporal.io/cloud/tcld).
-- Use open-source tools like [certstrap](https://github.com/square/certstrap).
+- Use Temporal's built-in certificate generation tool [`tcld`](https://docs.temporal.io/cloud/tcld) to create a `.pem` file containing the certificate.
+- Use open-source tools like [Certstrap](https://github.com/square/certstrap).
+  The sample below creates a `.crt` certificate file with Certstrap, but the tool can also output `.pem` format like `tcld`.
```

Comment:
Let's avoid "below" and make it more active voice.  The sentence that follows says a similar thing too - that you'll use certstrap.

Reply:
Will fix.